ibridges/data_operations.py

```diff
@@ -274,10 +287,19 @@ def _create_local_dest(session: Session, irods_path: IrodsPath, local_path: Path
         source_to_dest.append((cur_ipath, cur_lpath))
     return source_to_dest
 
+def _create_local_subtree(session: Session, irods_path: IrodsPath, local_path: Path):
+
+    coll = get_collection(session, irods_path)
+    subcolls = _get_subcoll_paths(session, coll)
+    # create all folders from collections including empty ones
+    folders = [local_path.joinpath(coll.name, *sc.relative_to(irods_path).parts)
+               for sc in subcolls]
+    for folder in folders:
+        folder.mkdir(parents=True, exist_ok=True)
 
 def _download_collection(session: Session, irods_path: Union[str, IrodsPath], local_path: Path,
                          overwrite: bool = False, ignore_err: bool = False, resc_name: str = '',
-                         options: Optional[dict] = None):
+                         copy_empty_folders: bool = False, options: Optional[dict] = None):
```

Comment:
Is there a reason why the default value is False here and True in the `download` function? I would suggest putting everything at True by default if not.

Reply:
Good catch ... yes, the default values should be the same and all default to `True`